src/NetTopologySuite/Shape/CubicBezierCurve.cs

```diff
@@ -82,7 +82,7 @@ public static Geometry Create(Geometry geom, Geometry controlPoints)
         }
 
         private readonly double _minSegmentLength = 0.0;
-        private readonly int _numVerticesPerSegment = 16;
+        private static readonly int _numVerticesPerSegment = 16;
```

Comment:
Can we expose this as a static property and compute the interpolation parameters when it is changed?
Otherwise `const` and `NumVerticesPerSegment`